tests/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactoryTest.php

```diff
@@ -305,7 +305,7 @@ public function testCreateWithPropertyRegexRestriction(): void
         $schema = $validationPropertyMetadataFactory->create(DummyValidatedEntity::class, 'dummy')->getSchema();
         $this->assertNotNull($schema);
         $this->assertArrayHasKey('pattern', $schema);
-        $this->assertEquals('dummy', $schema['pattern']);
+        $this->assertEquals('^(?:dummy)$', $schema['pattern']);
```

Comment:
note: it would be nicer to get `'^dummy$'` here, but in general the wrapped part could be anything...